### PR TITLE
Several things (fixes and feat)

### PR DESCRIPTION
**Pull request by jp-rl**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/modal/confirmDialog/ConfirmService.cs [hunk 1]
```diff
@@ -22,20 +22,23 @@ public class ConfirmService
...
+        /// <param name="autoFocusButton">the autofocus button</param>
         /// <returns></returns>
         public async Task<ConfirmResult> Show(
             OneOf<string, RenderFragment> content,
             OneOf<string, RenderFragment> title,
             ConfirmButtons confirmButtons,
             ConfirmIcon confirmIcon,
-            ConfirmButtonOptions options)
+            ConfirmButtonOptions options, 
+            ConfirmAutoFocusButton autoFocusButton = ConfirmAutoFocusButton.Ok)
```
Comment (ElderJames) [inline components/modal/confirmDialog/ConfirmService.cs:33]: Nullable support is better, since focus may not be required.

**Review by ElderJames — COMMENTED** [review]
Thanks for contribute this @jp-rl

File: components/modal/confirmDialog/ConfirmService.cs (hunk 1 above)
Comment (ElderJames) [reply components/modal/confirmDialog/ConfirmService.cs:33]: Yes,Change to 
```suggestion
            ConfirmAutoFocusButton? autoFocusButton = ConfirmAutoFocusButton.Ok)
```

**Comment by jp-rl** [comment]
I have to change code after add Nullable to ConfirmAutoFocusButton.

**Review by ElderJames — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**